### ISSUE TITLE
Option for make tab to indent in Livescript.net

### ISSUE THREAD

**Issue by ceremcem** [gkz/LiveScript#896]
I use Livescript.net heavily when I want to show something to a friend. Since I can not use TAB to indent a block, indenting with space bar becomes frustrating. Can we place a checkbox in order to switch to a mode that will indent selected block when we press TAB button?

**Comment by summivox** [comment]
IIRC someone mentioned about having ACE in livescript.net before, but the idea was rejected then because a maintainer (forgot who) wanted to keep mobile download size lean.

That said, I am thinking about having an optional ACE editor that loads on demand.

**Comment by ceremcem** [comment]
Yes, I recall a conversation something like that. But in this case I only propose to have only TAB to indent... That won't add extra load except for a few bytes. 

ACE editor that loads on demand would be another great option.

**Comment by igl** [comment]
I think there was a pull request which was rejected by gkz because it had a copy of the whole ACE repo in it. Doing it a leaner integration with the compiled ace.js might get accepted.
